javascript/javascript1/week3/main3.js

```diff
@@ -0,0 +1,126 @@
+// Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+
+const nameToRemove = "Ahmad";
+for (let i = 0; i < names.length; i++) {
+
+  if (names[i] === nameToRemove) {
+
+    names.splice(i, 1);
+  }
+}
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+
+//  When will we be there
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function getTravelTime(object) {
+  let arrivalTime = (object.destinationDistance / object.speed) * 60;
+  let hours = Math.floor(arrivalTime / 60);
+  let minutes = Math.floor(arrivalTime % 60);
+  return hours + " hours and " + minutes + " minutes";
+}
```

Comment:
Overall good. But remember to look into the difference between let and const. If a variable does not change in it's lifetime, it better to use const.